global_schema.ddl

```diff
@@ -132,88 +128,143 @@ CREATE TABLE IF NOT EXISTS controller_node (
 		REFERENCES	controller(uuid)
 );
 
+/*
+ * External Controllers
+ */
+CREATE TABLE IF NOT EXISTS external_controller (
+	uuid			TEXT PRIMARY KEY,
+	controller_uuid	TEXT, -- TODO: Investigate this field
```

Comment:
We can drop this and assume that the primary key is the controller UUID. Affects the constraint below.